src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -122,11 +131,20 @@ class CombatLogParser extends MainCombatLogParser {
       });
     }
 
+    if (spiritBombUptimePercentage < 1.0 && this.selectedCombatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id)) {
+      results.addIssue({
+        issue: <span>Try to cast <SpellLink id={SPELLS.SPIRIT_BOMB_TALENT.id} /> more often. This is your core healing ability. Try to refresh it even if you have just one <SpellLink id={SPELLS.SOUL_FRAGMENT.id} /> available.</span>,
+        stat: <span>{Math.round(spiritBombUptimePercentage * 100)}% <SpellLink id={SPELLS.FRAILTY_SPIRIT_BOMB_DEBUFF.id} /> debuff total uptime (100% uptime is recommended) </span>,
+        icon: SPELLS.FRAILTY_SPIRIT_BOMB_DEBUFF.icon,
+        importance: getIssueImportance(spiritBombUptimePercentage, 0.90, 0.80),
+      });
+    }
+
     const castEfficiency = getCastEfficiency(CPM_ABILITIES, this);
```

Comment:
Please move this to a new CastEfficiency module, see the HolyPaladin spec for an example (it's fairly trivial, mostly a matter of copying an example, moving the 2 arrays and adding the module to your specModules).